Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.yml

```diff
@@ -3611,6 +3616,3088 @@ script:
+      description: Vendor-specific information about the security result priority.
+      type: String
+    - contextPath: GoogleChronicleBackstory.CuratedRuleDetections.collectionElements.references.securityResult.ruleId
+      description: Identifier for the security rule.
+      type: String
+    - contextPath: GoogleChronicleBackstory.CuratedRuleDetections.collectionElements.references.securityResult.ruleName
+      description: Name of the security rule.
+      type: String
+    - contextPath: GoogleChronicleBackstory.CuratedRuleDetections.collectionElements.references.securityResult.severity
+      description: Severity of a security event as estimated by the product vendor using values defined by the Chronicle UDM.
+      type: String
+    - contextPath: GoogleChronicleBackstory.CuratedRuleDetections.collectionElements.references.securityResult.severityDetails
+      description: Severity for a security event as estimated by the product vendor.
+      type: String
+    - contextPath: GoogleChronicleBackstory.CuratedRuleDetections.collectionElements.references.securityResult.threatName
+      description: Name of the security threat.
+      type: String
+    - contextPath: GoogleChronicleBackstory.CuratedRuleDetections.collectionElements.references.securityResult.urlBackToProduct
+      description: URL to direct you to the source product console for this security event.
+      type: String
+    - contextPath: GoogleChronicleBackstory.Token.name
+      description: The name of the command to which the value of the nextPageToken corresponds.
+      type: String
+    - contextPath: GoogleChronicleBackstory.Token.nextPageToken
+      description: A page token that can be provided to the next call to view the next page of detections. Absent if this is the last page.
+      type: String
+  - arguments:
+    - default: false
+      description: 'The value of the start time for your request. The format of Date should comply with RFC 3339 (e.g. 2023-01-02T15:00:00Z) or relative time. If not supplied, the product considers UTC time corresponding to 3 days earlier than the current time. Formats: YYYY-MM-ddTHH:mm:ssZ, YYYY-MM-dd, N days, N hours. If the date is supplied in duration, it will be calculated as time.now() - duration. Example: 2023-04-25T00:00:00Z, 2023-04-25, 2 days, 5 hours, 01 Mar 2023, 01 Feb 2023 04:45:33, 15 Jun.'
+      isArray: false
+      name: start_time
+      required: false
+      secret: false
+    - default: false
+      description: 'The value of the end time for your request. The format of Date should comply with RFC 3339 (e.g. 2023-01-02T15:00:00Z) or relative time. If not supplied, the product considers current UTC time. Formats: YYYY-MM-ddTHH:mm:ssZ, YYYY-MM-dd, N days, N hours. If the date is supplied in duration, it will be calculated as time.now() - duration. Example: 2023-04-25T00:00:00Z, 2023-04-25, 2 days, 5 hours, 01 Mar 2023, 01 Feb 2023 04:45:33, 15 Jun.'
```

Comment:
```suggestion
      description: 'The value of the end time for your request. The date format should comply with RFC 3339 (e.g., 2023-01-02T15:00:00Z) or relative time. If not supplied, the product considers current UTC time. Formats: YYYY-MM-ddTHH:mm:ssZ, YYYY-MM-dd, N days, N hours. If the date is supplied in duration, it will be calculated as time.now() - duration. Example: 2023-04-25T00:00:00Z, 2023-04-25, 2 days, 5 hours, 01 Mar 2023, 01 Feb 2023 04:45:33, 15 Jun.'
```